themes/socialblue/color/preview.js

```diff
@@ -64,10 +64,35 @@
       $colorPreview.find('.body-text a:not(.btn)').css('color', $colorPalette.find('input[name="palette[brand-link]"]').val());
 
       // Hero toggle background.
-      $colorPreview.find('.color-preview-hero .switch').once().click(function() {
-        $('.color-preview-hero .switch .lever').toggleClass("lever-on");
-        $('.color-preview-hero .cover').toggleClass("cover-img")
-          .toggleClass("cover-img-gradient");
+      var localStorage =  window.localStorage;
+      var colorPreviewLever = $('.color-preview-hero .switch .lever');
+      var colorPreviewCover = $('.color-preview-hero .cover');
+
+      if (localStorage.getItem('lever-on') &&
+        localStorage.getItem('set-settings-theme')) {
+          colorPreviewLever.addClass('lever-on');
+          colorPreviewCover.addClass('cover-img')
+            .addClass('cover-img-gradient');
```

Comment:
We can move this in with the previous `addClass` call, no need for two separate function calls.